mlflow/sagemaker/__init__.py

```diff
@@ -126,16 +126,15 @@ def push_image_to_ecr(image=DEFAULT_IMAGE_NAME):
     # x = ecr_client.get_authorization_token()['authorizationData'][0]
     # docker_login_cmd = "docker login -u AWS -p {token} {url}".format(token=x['authorizationToken']
     #                                                                ,url=x['proxyEndpoint'])
+
+    docker_login_cmd = "aws ecr get-login-password" \
```

Comment:
Was this tested in a windows command prompt or the windows linux subsystem?

Reply:
It was tested from the windows command prompt